gulp/webpack.js

```diff
@@ -0,0 +1,14 @@
+var gulp = require('gulp');
+
+var run = require('gulp-run');
+
+
+gulp.task('webpack', function (cb) {
+  run('webpack -d').exec()
```

Comment:
the watcher in webpack is much more efficient that if we used gulp and re-ran the whole thing.